ch.bailu.gtk_meteo.json

```diff
@@ -0,0 +1,51 @@
+{
+  "app-id": "ch.bailu.gtk_meteo",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk17"
+  ],
+  "command": "gtk-meteo.sh",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--env=GIO_EXTRA_MODULES=/app/lib/gio/modules",
+    "--env=JAVA_HOME=/app/jre",
+    "--env=LIBO_FLATPAK=1",
+    "--env=PATH=/app/bin:/app/jre/bin"
```

Comment:
```suggestion
    "--env=PATH=/app/bin:/usr/bin:/app/jre/bin"
```

Reply:
You are removing everything in runtime from PATH, this going to break massively.